src/showcase/src/backgrounds.ts

```diff
@@ -1,3 +1,8 @@
-import { loadIdentityBackground } from "$src/components/identityCard";
+import { IdentityBackground } from "$src/components/identityCard";
+import { ImageMetadata } from "astro";
+import backgroundImage from "./assets/identityCard.png";
 
-export const identityBackground = loadIdentityBackground();
+export const identityBackground = IdentityBackground.getSingleton(
+  // When running the build, this is also build but not in an Astro envioronment
+  (backgroundImage as unknown as ImageMetadata).src
```

Comment:
why is the casting necessary?

Reply:
because when building II it also check this import, whose time is not of ImageMetadata from Astro.

Once we move this to its own module maybe we'll be able to remove the casting.